src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from './components/App';
 
+// ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
- [ ]  Please kindly remove this commented line of code to make your code clean according to practices. 👍

Reply:
Thank you so much for your keen observation